packages/react-components/src/components/iot-app-kit-config/defaultValues.ts

```diff
@@ -2,7 +2,7 @@ import { FEATURE_FLAGS } from './types';
 
 const DEFAULT_FEATURE_CONFIG = {
   [FEATURE_FLAGS.ENABLE_E_CHARTS]: false,
-  [FEATURE_FLAGS.ENABLE_SYMBOL_LIBRARY]: false,
+  [FEATURE_FLAGS.ENABLE_SYMBOL_LIBRARY]: true,
```

Comment:
Also, please start using the ```getConfigValue``` from the react-component ... instead of the iot-app-config HOC/provider, we will be removing it soon

Reply:
left that feature flag true just so i can test the feature. followed up with you on slack regarding your second comment.